beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -1,56 +1,56 @@
 ---
-title: Add React to a Website
+title: किसी वेबसाइट पर React जोड़ें
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption, and you can use as little or as much React as you need. Whether you're working with micro-frontends, an existing system, or just giving React a try, you can start adding interactive React components to an HTML page with just a few lines of code—and no build tooling!
+React को शुरू से ही धीरे-धीरे अपनाने के लिए डिज़ाइन किया गया है, और आप जितना चाहें उतना कम या ज्यादा React का उपयोग कर सकते हैं। चाहे आप माइक्रो-फ्रंटेंड, एक मौजूदा सिस्टम के साथ काम कर रहे हों, या सिर्फ React को आज़मा रहे हों, आप HTML पेज पर इंटरेक्टिव React कौम्पोनॅन्ट को कोड की कुछ पंक्तियों के साथ जोड़ना शुरू कर सकते हैं - और कोई बिल्ड टूलिंग नहीं!
```

Comment:
`और कोई बिल्ड टूलिंग नहीं!` => `और कोई बिल्ड टूलिंग की ज़रूरत नहीं!`

I am not very satisfied with this suggestion also. Let me know if you have any other suggestion for this

Reply:
I think your suggestion is right as we need to express our stress on that part.